server/models/historical_meteorological.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Float, Integer, String
+
+from server.db.base_class import Base
+
+
+class Historical_Meteorological(Base):
+    __tablename__ = "Historical Meteorological Data"
+
+    id = Column(Integer, index=True, nullable=False, primary_key=True)
+    station_id = Column(String(10), index=True, nullable=False, unique=False)
+    year = Column(String, nullable=False)
+    month = Column(String, nullable=False)
+    day = Column(String, nullable=False)
+    hour = Column(String, nullable=False)
+    minute = Column(String, nullable=False)
+    wind_direction = Column(Integer, nullable=True)
+    wind_speed = Column(Float, nullable=True)
+    wind_gust = Column(Float, nullable=True)
+    wave_height = Column(Float, nullable=True)
+    dominant_period = Column(Integer, nullable=True)
+    average_wave_period = Column(Float, nullable=True)
+    mean_wave_direction = Column(String, nullable=True)
+    air_pressure = Column(Float, nullable=True)
+    pressure_tendency = Column(Float, nullable=True)
+    air_temperature = Column(Float, nullable=True)
+    water_temperature = Column(Float, nullable=True)
+    dewpoint = Column(Float, nullable=True)
+    salinity = Column(Float, nullable=True)
+    visibility = Column(Float, nullable=True)
+    tide = Column(Float, nullable=True)
```

Comment:
Why did you choose to set the weather related data columns as nullable?

Reply:
You'd be surprised but they don't always populate the weather fields. Sometimes it's just null. Also iirc those 4 columns were all null when I was cross referencing building out the models.

Reply:
Whenever it's null, is it represented by this string: `"MM"`?

Reply:
Yes. Same principle goes for wave data as well.